CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [Stable release 5.2.1] - 2024-05-28
+
+This update brings the below changes over the previous release:
+
+### Changed
+
+- Upgraded `Azure.Identity` version from 1.10.3 to 1.11.3 https://github.com/dotnet/SqlClient/pull/2528
+- Code Health improvements: https://github.com/dotnet/SqlClient/pull/2467
```

Comment:
follow markup language for all links. you can see samples at [here](https://github.com/dotnet/SqlClient/pull/2370/files).